src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/createExpandedColumnHeaders.test.ts

```diff
@@ -0,0 +1,1062 @@
+import Header from '../../components/utils/Header';
+import createExpandedColumnHeaders from '../createExpandedColumnHeaders';
+// these tests cover the test cases for headers that used to be in MultiHeaderDataTable
+
+describe('createExpandedColumnHeaders', () => {
+  test('should return an empty array if no column headers are provided', () => {
+    expect(createExpandedColumnHeaders([])).toEqual([]);
+  });
+
+  test('should return a single row of column headers if no groups are provided', () => {
+    const columnHeaders: Header[] = [
+      new Header('1', '1'),
+      new Header('2', '2'),
+      new Header('3', '3'),
+    ];
+
+    expect(createExpandedColumnHeaders(columnHeaders)).toEqual([
```

Comment:
Same as for `createExpandedRowColumnHeaders`, could add a type parameter here on the `toEqual` to get compile time feedback about type problems.

Same applies for rest of test cases